package/gluon-hoodselector/Makefile

```diff
@@ -0,0 +1,24 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-hoodselector
+
+GLUON_VERSION:=3
+PKG_VERSION:=2
+
+include ../gluon.mk
+
+define Package/gluon-hoodselector
+  TITLE:=Automatically migrate nodes between domains.
+  DEPENDS:=+luaposix +libgluonutil +lua-math-polygon +libjson-c +gluon-site +micrond +luabitop @GLUON_MULTIDOMAIN
+  CONFLICTS:=+gluon-config-mode-domain-select
+endef
+
+define Package/gluon-hoodselector/description
+  The purpose of Hoodselector is to automatically detect in which domain
```

Comment:
Hoodselector automatically detects ... based on its geolocation. Domains require bounding boxes defined as polygons or rectangles. Hoodselector selects a domain from the list of known domains and migrates towards it ...

Reply:
fixed